pkg/sql/create_type.go

```diff
@@ -99,6 +101,91 @@ func getCreateTypeParams(
 	return typeKey, id, nil
 }
 
+// When a type is created in Postgres, Postgres will implicitly create an array
+// type of that user defined type. This array type tracks changes to the
+// original type, and is dropped when the original type is dropped.
+// createArrayType creates the implicit array type for the input TypeDescriptor
```

Comment:
nit: createArrayType should be the first word in this comment block

Reply:
done